Userland/Utilities/CMakeLists.txt

```diff
@@ -9,7 +9,6 @@ set(CMD_SOURCES
     isobmff.cpp
     js.cpp
     lzcat.cpp
-    shred.cpp
```

Comment:
btw I found this file isn't actually used at all, utilities come from Meta/Lagom/CMakeLists.txt
(I was wondering why my sql PR didn't complain about missing sql.cpp when I was rebasing)